lib/opensusebasetest.pm

```diff
@@ -881,6 +881,9 @@ sub wait_boot {
         #
         $self->handle_pxeboot(bootloader_time => $bootloader_time, pxemenu => 'pxe-custom-kernel', pxeselect => 'pxe-custom-kernel-selected');
     }
+    # When no bounce back on power KVM, we need skip bootloader process and go ahead when 'displaymanager' matched.
+    elsif (get_var('OFW') && (check_screen('displaymanager', 5))) {
```

Comment:
why don't you put the check_screen inside the `elsif` ? otherwise you have an empty block.

Reply:
I just hope won't affect other conditions, if put the check_screen inside elsif, when check_screen failed, there is no chance to run other if branches. So just see the condition of OFW and check_screen as a whole condition and won't affect other conditions to be checked when it failed.